api/users.go

```diff
@@ -9,17 +9,20 @@ import (
 )
 
 const (
-	userPath                   = "/auth/admin/realms/:realm/users"
-	usersAdminExtensionAPIPath = "/auth/realms/:realmReq/api/admin/realms/:realm/users"
-	userCountPath              = userPath + "/count"
-	userIDPath                 = userPath + "/:id"
-	userGroupsPath             = userIDPath + "/groups"
-	userGroupIDPath            = userGroupsPath + "/:groupId"
-	executeActionsEmailPath    = userIDPath + "/execute-actions-email"
-	sendReminderEmailPath      = "/auth/realms/:realm/onboarding/sendReminderEmail"
-	smsAPI                     = "/auth/realms/:realm/smsApi"
-	sendSmsCode                = smsAPI + "/sendNewCode"
-	shadowUser                 = userIDPath + "/federated-identity/:provider"
+	userPath                       = "/auth/admin/realms/:realm/users"
+	adminExtensionAPIPath          = "/auth/realms/:realmReq/api/admin/realms/:realm"
+	usersAdminExtensionAPIPath     = adminExtensionAPIPath + "/users"
+	sendEmailAdminExtensionAPIPath = adminExtensionAPIPath + "/send-email"
+	sendSMSAdminExtensionAPIPath   = adminExtensionAPIPath + "/send-sms"
```

Comment:
Specifications in JIRA ticket were not correctly defined:
POST /auth/realms/<realm>/sendSms